contracts/MetadataViews.cdc

```diff
@@ -129,4 +129,55 @@ pub contract MetadataViews {
             return "ipfs://".concat(self.cid)
         }
     }
+
+
+    // A view to represent a File that is an image
+    pub struct Image {
+        pub let file: AnyStruct{File}
+
+        init(_ file: AnyStruct{File}) {
+          self.file=file
+        }
+    }
+
+
+    // A view to represent a File that is video
+    pub struct Video {
+        pub let file: AnyStruct{File}
+
+        init(_ file: AnyStruct{File}) {
+          self.file=file
+        }
+    }
+
+    // A view to represent a File that is audio
+    pub struct Audio {
+        pub let file: AnyStruct{File}
+
+        init(_ file: AnyStruct{File}) {
+          self.file=file
+        }
+    }
+
+    // A license according to https://spdx.org/licenses/
+		//
+		//This view can be used if the content of an NFT is licensed. 
+    pub struct License {
+        pub let spdxIdentifier: String
+
+        init(_ identifier: String) {
+            self.spdxIdentifier = identifier
+        }
+    }
+
+    //A view to expose an url to this item on an external site
+
+		//This can be used in solutions like .find/blocto to know what should happen when you click on a link
```

Comment:
```suggestion
    // A view to expose a URL to this item on an external site.
    //
    // This can be used by applications like .find and Blocto to direct users to the original link for an NFT.
```